src/app/ar-assets/_components/ArAssetItem.tsx

```diff
@@ -3,19 +3,23 @@ import { Dto_ArAssetsResponse } from '@/api/@types';
 import { Button } from '@/shared/components/common/Button';
 import { Divider } from '@/shared/components/common/Divider';
 import { Image } from '@/shared/components/common/Image';
-import { Card, Flex } from '@/shared/components/common/Layout';
+import { Card, Flex, Stack } from '@/shared/components/common/Layout';
 import { QRCode } from '@/shared/components/common/QRCode';
+import { Radio } from '@/shared/components/common/Radio';
 import { Text } from '@/shared/components/common/Text';
 import { Title } from '@/shared/components/common/Title';
 import { getQRCodeUrl } from '@/shared/components/features';
 import { IconPencil, IconSpeakerphone } from '@/shared/components/icons';
 import { ROUTES } from '@/shared/constants';
+import { useForm } from '@/shared/hooks/useForm';
 
 type Props = {
   arAsset: Dto_ArAssetsResponse;
+  hideElement?: boolean;
 };
 
-export const ArAssetItem = ({ arAsset }: Props) => {
+export const ArAssetItem = ({ arAsset, hideElement }: Props) => {
```

Comment:
```suggestion
export const ArAssetItem = ({ arAsset, hideElement = false }: Props) => {
```
こんな感じで初期値を設定できるはずだからわかりやすくするために